build/configuration/pom.xml

```diff
@@ -192,7 +192,7 @@
       <version.ow2.asm>9.7</version.ow2.asm>
       <version.picketbox>5.0.3.Final</version.picketbox>
       <version.picketlink>2.5.5.SP12</version.picketlink>
-      <version.protostream>5.0.5.Final</version.protostream>
+      <version.protostream>5.0.6-SNAPSHOT</version.protostream>
```

Comment:
Can you rebase this PR to pick up ProtoStream 5.0.7.Final which contains your fix ?

Reply:
Sure!